src/main/java/com/contrastsecurity/exceptions/ContrastException.java

```diff
@@ -0,0 +1,15 @@
+package com.contrastsecurity.exceptions;
+
+/** Generic {@link RuntimeException} thrown by Contrast code. */
+public class ContrastException extends RuntimeException {
```

Comment:
We have this same wrapped RuntimeException in the java-agent codebase. Is this a common pattern? It makes more sense for the agent in some ways because it makes it apparent for the users that the issue is one with Contrast, but for the sdk I feel like `HttpResponseException` could just extend `RuntimeException` given the scope of this project. Do other SDKs wrap their exceptions in a way like this?

Reply:
> Do other SDKs wrap their exceptions in a way like this?

yes. examples:

* https://azure.github.io/azure-sdk/java_introduction.html
* https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/exception/SdkException.html
* https://javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.13.0-rc1/com/fasterxml/jackson/core/JacksonException.html